tests/integration/test_comb.js

```diff
@@ -24,58 +23,86 @@ async function create_page ( url ) {
     return page;
 }
 
+class PageTestUtils {
+    constructor( page ) {
+	this.returnPageError		= () => page.on('pageerror', async error => {
```

Comment:
Can you rename this because the word "return" does not describe it accurately?  Perhaps `logPageErrors`